kotlin-inject-runtime/src/commonMain/kotlin/me/tatarka/inject/annotations/Annotations.kt

```diff
@@ -32,4 +32,4 @@ annotation class Assisted
 annotation class Qualifier
 
 @Target(FUNCTION)
-annotation class TargetComponentAccessor
+annotation class KmpComponentCreator
```

Comment:
I like `KmpComponentProvider` better, but I didn't want to cause any confusion with `@Provides`

Reply:
Yeah, that's a better name! Curious what you'd think about `@CreateKmpComponent`? Feel like it reads slightly better but open to feedback.

Reply:
I think both work. It's action vs descriptor, i.e. verb vs noun.

Do you want to make the call?